pkg/apis/operator/v1alpha1/validation/garden_test.go

```diff
@@ -1300,7 +1300,7 @@ var _ = Describe("Validation Tests", func() {
 			})
 
 			Context("ETCD", func() {
-				It("should complain if both bucket name and provider config are set", func() {
+				It("should allow provider config even if bucket name is set", func() {
```

Comment:
Is this test really needed now?

Reply:
Absolutely – since we now allow setting both `bucketName` and `providerConfig`, we should add an explicit unit test for this scenario to ensure it passes validation.